### DISCUSSION TITLE
Zim 0.75.2 released!

### DISCUSSION THREAD

**Discussion by jaap-karssenberg** [zim-desktop-wiki/zim-desktop-wiki#2410] [Announcements]
Zim 0.75.2 has just been released. This is an update with all the incremental improvements over the past 3/4 year.

##  0.75.2 - Thu 06 Jul 2023
* Add pagename completion by words in "Jump to"
* Let tasklist window remember selection state
* Let tasklist window use a search bar
* Add count in tasklist window
* Fix auto-collapse in page index for child nodes
* Fix import of gtkspellcheck
* Make spellcheck more robust for page reload
* Fix bug in insert screenshot
* Add scrollbar to custom tool window
* Fix config dirs for portable install
* Use environment variable for `XDG_TEMPLATES_DIR`
* Fix links to anchors on export
* Fix bug in copy-paste of interwiki links
* Updated diagrameditor and sequencediagrameditor plugins to generate SVG
* Add keybinding for cutting the current line(s)
* Fix fullscreen headerbar visibility

**Comment by supertoothy** [comment]
Thank you Jaap!
